refgenconf/refgenconf.py

```diff
@@ -323,7 +328,9 @@ def list_local(self, genome=None, order=None):
         :return str, str: text reps of locally available genomes and assets
         """
         if genome is not None:
-            _assert_gat_exists(self[CFG_GENOMES_KEY], gname=genome)
+            if isinstance(genome, str):
+                genome = [genome]
+            [_assert_gat_exists(self[CFG_GENOMES_KEY], gname=g) for g in genome]
```

Comment:
This is strange syntax, like it's accumulating a result but neither using nor storing it. Is `_assert_gat_exists` called just for effects(s)? If so, this would be more readable as a `map` or a `for` loop. An even better approach may be to accumulate all the failures, failing with a message for all of them, so that the user's aware of more than one problem without needing to repeatedly retry some operation.

Reply:
agreed, list comprehension was not the best choice here. thanks!